app/views/users/profile.html.erb

```diff
@@ -107,25 +101,38 @@
 
     <div class="profile_icons d-flex justify-content-center" style="gap: 8px">
       <% if @twitter.nil? == false %>
+      <span>
+       <a aria-label="Delete tag" data-confirm="Are you sure you want to delete it?" class="tag-delete" data-remote="true" href="/profile/tags/delete/<%= @profile_user.id %>?name=twitter:<%= @twitter_user %>" data-method="delete"><span style=" color: white; position: absolute; display: block; " class="badge badge-primary mb-1">x</span></a>         
```

Comment:
The lines of code that contain the delete link are very long, do you think you could format them so it's more readable? Thanks!

Reply:
okay